llvm/lib/Bitcode/Reader/BitcodeReader.cpp

```
@@ -1144,6 +1144,22 @@ static bool getDecodedDSOLocal(unsigned Val) {
   }
 }
 
+static CodeModel::Model getDecodedCodeModel(unsigned Val) {
+  switch (Val) {
+  case 1:
+    return CodeModel::Tiny;
+  default: // Map unknown values to small.
```

should error out on invalid value